utils/migration_tools/from_legacy_neptune/README.md

```diff
@@ -0,0 +1,139 @@
+| `--new-project` | No | Name of the new project in the format `PROJECT_NAME`. The project will be created if it doesn't already exist. If not provided, the project name will be the same as the legacy project name. |
+| `--max-workers` | No | Maximum number of parallel workers to use for copying runs. Defaults to ThreadPoolExecutor’s default. See [ThreadPoolExecutor](https://docs.python.org/3/library/concurrent.futures.html#concurrent.futures.ThreadPoolExecutor) for details. |
+
+---
+
+## How it works
+- The target project is created in the new workspace if it doesn't already exist.
+  - Metrics and parameters are copied from memory
+    - The `monitoring/` namespace in the source run is copied to the `runtime/` namespace in the target run.
+    - Most `sys` attributes are copied to the `legacy_sys` namespace. For details, see [Notes and Limitations](#notes-and-limitations).
+  - Files are first downloaded to a temporary directory and then uploaded to the new run.
+    - Temporary files are stored in a `.tmp_*` directory for troubleshooting. You can delete this folder after verifying the migration. Ensure that you have sufficient space to store the temporary files.
+  - Unsupported metadata (see Notes and Limitations) is skipped.
+
+---
+
+## Notes and limitations
+- Runs can be copied only from one project at a time.
+- If a new project name isn't provided and a project with the same key as the source project already exists in the new workspace, the script will silently fail.  
+- Avoid creating new runs in the source project while the script is running as these might not be copied.
+- Timestamp values appended to each step of series metrics are in the local timezone of the script execution environment. This can lead to variations between the source and target run charts if the X-axis is set to relative time and the source run was created in a different timezone.
+
+### Data that isn't copied in the same format
+- If the source run doesn't have a `custom_run_id`, the source run ID (`sys/id`) will be used instead as the `run_id` of the target run
+- The `monitoring` namespace in the source run is copied to the `runtime` namespace in the target run
+- All `sys` attributes except `state`, `description`, `name`, `custom_run_id`, `tags`, and `group_tags` are copied to the `legacy_sys` namespace
+- `sys/running_time` and `sys/monitoring_time` are copied to `legacy_sys/running_time` and `legacy_sys/monitoring_time` respectively as floats in seconds
+- `sys/size` is copied to `legacy_sys/size` as an integer in bytes
+
+### Data that isn't copied
+- Project description and members
+- Project and model metadata
+- Artifacts*
+- FileSet (including source code)*
+- All FileSeries objects*
+  - Once FileSeries are supported in Neptune 3.x, the custom steps and descriptions of files in FileSeries still won't be copied
+  - Run state: `sys/state`
+  - Git info: `source_code/git`
+
+<sup>*</sup> Support for these will be added in upcoming releases.
```

Comment:
(nit) Probably no need to apply superscript to the asterisk?

```suggestion
\* Support for these will be added in upcoming releases.
```

Reply:
I forgot that escaping exists 🙈